### PR TITLE
Add integration test

### PR DESCRIPTION
**Pull request by HelenMel**
__WHY__:
There will be 2 features, that would be very hard to test manually

__WHAT__:
- Added localstack to test S3
- Added single basic integration test 
- Added Integration tests README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -207,6 +207,26 @@ timestamp.timezone=Europe/Berlin
...
+### Integration testing
+
+Integration tests are implemented using JUnit, Gradle and Docker.
+
+To run them, you need:
+- Docker installed.
+
+Integration testing doesn't require a valid AWS credentials.
+
+To simulate AWS S3 behaviour tests uses [LocalStack](https://github.com/localstack/localstack-java-utils) provided by AWS.
```
Comment (ivanyu) [inline README.md:221]: Localstack is not really provided by AWS
```suggestion
To simulate AWS S3 behaviour tests uses [LocalStack](https://github.com/localstack/localstack-java-utils).
```

File: src/integration-test/java/io/aiven/kafka/connect/KafkaIntegrationBase.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    default Map<String, String> basicConnectorConfig(final String connectorName) {
+        final Map<String, String> config = new HashMap<>();
+        config.put("name", connectorName);
+        config.put("key.converter", "org.apache.kafka.connect.converters.ByteArrayConverter");
+        config.put("value.converter", "org.apache.kafka.connect.converters.ByteArrayConverter");
+        config.put("tasks.max", "1");
+        return config;
+    }
+
+    static void startTestingProcess(File pluginDir) throws IOException, InterruptedException {
```
Comment (ivanyu) [inline src/integration-test/java/io/aiven/kafka/connect/KafkaIntegrationBase.java:82]: `startTestingProcess` doesn't seem much corresponding to what's going on in the function. Maybe rename it to something like `extractConnectorPlugin` or the like?

File: src/integration-test/java/io/aiven/kafka/connect/IntegrationTest.java [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@ExtendWith(LocalstackDockerExtension.class)
+@LocalstackDockerProperties(services = {"s3"})
+@Testcontainers
+final class IntegrationTest implements KafkaIntegrationBase {
+    private static final String S3_ACCESS_KEY_ID = "test-key-id0";
+    private static final String S3_SECRET_ACCESS_KEY = "test_secret_key0";
+    private static final String TEST_BUCKET_NAME = "olena-test";
```
Comment (ivanyu) [inline src/integration-test/java/io/aiven/kafka/connect/IntegrationTest.java:66]: Maybe a bit less personalized bucket name 🙂

File: README.md (hunk 1 above)
Comment (ivanyu) [inline README.md:221]:
```suggestion
To simulate AWS S3 behaviour the tests use [LocalStack](https://github.com/localstack/localstack-java-utils) provided by AWS.
```

File: README.md [hunk 4]
```diff
@@ -207,6 +207,26 @@ timestamp.timezone=Europe/Berlin
...
+## Development
+
+### Integration testing
+
+Integration tests are implemented using JUnit, Gradle and Docker.
+
+To run them, you need:
+- Docker installed.
+
+Integration testing doesn't require a valid AWS credentials.
```
Comment (ivanyu) [inline README.md:219]:
```suggestion
Integration testing doesn't require valid AWS credentials.
```

**Review by ivanyu — CHANGES_REQUESTED** [review]
Overall LGTM, some small comments.

File: README.md (hunk 1 above)
Comment (HelenMel) [reply README.md:221]: Indeed. Not yet ;)))
- [ ] Remove comment

File: src/integration-test/java/io/aiven/kafka/connect/KafkaIntegrationBase.java (hunk 2 above)
Comment (HelenMel) [reply src/integration-test/java/io/aiven/kafka/connect/KafkaIntegrationBase.java:82]: - rename

File: src/integration-test/java/io/aiven/kafka/connect/IntegrationTest.java (hunk 3 above)
Comment (HelenMel) [reply src/integration-test/java/io/aiven/kafka/connect/IntegrationTest.java:66]: Right)))
- [ ] Change bucket name

File: README.md (hunk 1 above)
Comment (HelenMel) [reply README.md:221]: How about without `the`, but with comma?
- [ ] fix grammar

File: README.md (hunk 4 above)
Comment (HelenMel) [reply README.md:219]: - [ ] Fix grammar

**Review by ivanyu — APPROVED** [review]

**Pull request merged**